### PR TITLE
Feature/on demand parking/qr

### PR DESCRIPTION
**Pull request by Yuichi-Kurino**
# Administrative Info

Make sure your branch name conforms to: `feature/hotfix/.../describe_your_branch_with_underscores

### Changes
- feature to download the image of qr code

## Checklist

Remove any items which are not applicable.

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added tested my program thoroughly
-   [ ] Did you add/sub fields in the environment file? if so please update .env_sample

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| [TODO Screenshot Title] |
| ----------------------- |
Screen Shot 2022-11-20 at 1 37 18 PM
Screen Shot 2022-11-20 at 1 37 47 PM

| [TODO Screenshot Link]  |

### CODE REVIEW

File: database/model/userModel.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const db = require('../dbConfig');
```
Comment (mingkaili) [inline database/model/userModel.js:1]: Can we change this file name to `adminModel.js`? Also, just curious, are you using this anywhere?

**Review by mingkaili — APPROVED** [review]
LGTM. If you are not using the admin stuff, go ahead and delete it.

File: database/model/userModel.js (hunk 1 above)
Comment (mingkaili) [reply database/model/userModel.js:1]: Friendly ping to handle this and then merge the PR.

Comment (Yuichi-Kurino) [reply database/model/userModel.js:1]: got it

**Pull request merged**